openapi/platform/platform.yaml

```diff
@@ -1112,7 +1112,191 @@ paths:
                       id:
                         type: string
                         example: AG-1234567
-                        description: The ID of a business location to grant default access to
+                        description: The ID of a business location to grant access to
+                      type:
+                        type: string
+                        description: This should always be `businessLocations`
+                        default: businessLocations
+                        example: businessLocations
+                    required:
+                      - id
+                      - type
+              required:
+                - data
+      responses:
+        '201':
+          description: Created
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/users'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+      x-lifecycle:
+        status: trustedTester
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Users can be granted access to business location(s) using the POST operation. After the operation completes, the user will have access to all their previous business locations plus any new locations in the POST request. If the user already has access to one or more of the specified locations they will be silently ignored.
+      parameters:
+        - schema:
+            type: string
+            example: Bearer <Access Token>
+            pattern: ^Bearer\s\S+
+          in: header
+          name: Authorization
+          description: A Bearer access token to identify the user the app is acting on behalf of. See the Authorization guide for details.
+          required: true
+        - schema:
+            type: string
+            default: application/vnd.api+json
+            enum:
+              - application/vnd.api+json
+          in: header
+          name: Content-Type
+          required: true
+          description: Indicates the format of the request body being sent. In most cases you will want `application/vnd.api+json`
+      security:
+        - OAuth2:
+            - user.admin
+      tags:
+        - Users
+    get:
```

Comment:
We are missing the `options` operation for the `/users/{id}/relationships/businessLocations` path. You can basically just copy the yaml from any of the other paths.